src/pages/diagram/dialogs/list-processes/list-processes.tsx

```diff
@@ -41,31 +50,59 @@ export const ListProcesses: React.FC<Props> = ({
   function getSubtitle(process: TProcess) {
     const createdAt = getLongFormatByDate(process.created_at);
     const { state, status } = process;
-
     return `${state.node_id} - ${status} - ${createdAt}`;
   }
 
   useEffect(() => {
     const request = async () => {
       const response = await listByWorkflowId(workflowId);
       setProcesses(response);
+      setFiltered(response);
     };
-
     if (isOpen) {
       request();
     }
   }, [isOpen, workflowId]);
 
+  function onFilter(payload: any) {
+    const filter = processes.filter((process) => {
+      const nodeId =
+        _isEmpty(payload.nodeId) ||
+        process.state.node_id.includes(payload.nodeId);
+
+      const status =
+        _isEmpty(payload.status) || process.status.includes(payload.status);
+
+      const before =
+        _isNull(payload.finalDate) ||
+        isBefore(new Date(process.created_at), payload.finalDate);

```

Comment:
Colocar o new Date() em uma const no inicio do método
```suggestion
        const createdAt = new Date(process.created_at);
        isBefore(createdAt, payload.finalDate);

```